docs/USAGE.md

```diff
@@ -58,7 +58,8 @@ Use these environment variables to configure the tracing library:
 | `OTEL_MAX_LOGFILE_SIZE` | The maximum size for tracer log files, in bytes. | `10 MB` |
 | `OTEL_TRACE_LOG_PATH` | The path of the profiler log file. | Linux: `/var/log/OTEL/dotnet/dotnet-profiler.log`<br>Windows: `%ProgramData%"\OTEL .NET Tracing\logs\dotnet-profiler.log` |
 | `OTEL_DIAGNOSTIC_SOURCE_ENABLED` | Enable to generate troubleshooting logs with the `System.Diagnostics.DiagnosticSource` class. | `true` |
-| `OTEL_DOTNET_TRACER_DISABLED_INSTRUMENTATIONS` | The instrumentations you want to disable, if any, separated by a comma. These are the supported integrations: `AspNet`, `HttpClient`, `SqlClient`, `MongoDb` |  |
+| `OTEL_DOTNET_TRACER_INSTRUMENTATIONS` | The instrumentations you want to enable, separated by a comma. These are the supported integrations: `AspNet`, `HttpClient`, `SqlClient`, `MongoDb` |  
+| `OTEL_DOTNET_TRACER_DISABLED_INSTRUMENTATIONS` | The instrumentations you want to disable, if any, separated by a comma.  |
```

Comment:
This does not state supported values. Also what happens if someone add the same instrumentation both here and in enabled variable?

Reply:
1. It can be used for disabling both SDK instrumentation and IL-rewrite instrumentations. So it could be whatever can be provided via `OTEL_DOTNET_TRACER_INSTRUMENTATIONS` or `integrations.json`.
2. It will be disabled

I will try to refine the docs

Reply:
Tried to address it here: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/204/commits/7271fc0aec0ac69fc954c9d3de92d53ed77b4f15